app/controllers/moderations_controller.rb

```diff
@@ -38,7 +38,11 @@ def comment
 
   def actions_panel
     load_article
-    render template: "moderations/actions_panel"
+    @author_flagged ||= Reaction.user_vomits.valid_or_confirmed.where(
+      user_id: session_current_user_id,
+      reactable_id: @moderatable.user_id,
+    ).any?
+    render template: "moderations/actions_panel", locals: { is_mod_center: params[:is_mod_center] }
```

Comment:
The `is_mod_center` param is used to selectively hide temporarily non-working Mod Panel buttons in the Mod Center context.